services/app-api/src/handlers/configuration.ts

```diff
@@ -82,4 +90,125 @@ async function getDBClusterID(secretName: string): Promise<string | Error> {
     return dbID
 }
 
-export { configurePostgres, getPostgresURL, getDBClusterID }
+async function configureEmailerFromDatabase(
+    store: Store
+): Promise<Omit<EmailConfiguration, 'stage' | 'baseUrl'> | Error> {
+    const emailSettings = await store.findEmailSettings()
+    if (emailSettings instanceof Error) {
+        return emailSettings
+    }
+
+    return {
+        emailSource: emailSettings.emailSource,
+        devReviewTeamEmails: emailSettings.devReviewTeamEmails,
+        helpDeskEmail: emailSettings.helpDeskEmail[0],
+        cmsReviewHelpEmailAddress: emailSettings.cmsReviewHelpEmailAddress[0],
+        cmsRateHelpEmailAddress: emailSettings.cmsRateHelpEmailAddress[0],
```

Comment:
Is there a reason why the db type on these is `TEXT[]`, but we're only reading out the first value?

Reply:
Yes - looks like it was to avoid cleanup. The types changed but we didn't have time to fully refactor so made a cleanup ticket -  [MCR-5018](https://jiraent.cms.gov/browse/MCR-5018): Clean up type drift between EmailSettingsType and EmailConfiguration type. We can do that after this ships but eventually this will handle multiple email address.